### DISCUSSION TITLE
Trouble with Icons in Next.js

### DISCUSSION THREAD

**Discussion by Nashtronaut** [React95/React95#371] [Q&A] [answered]
Hey Everyone, 

I apologize if this is a silly question. I have been using your project to create a personal website that I would be happy to share afterwards. Unfortunately, I have followed docs and story books to use Icons and I just can't seem to get them to work. 
 
```
import { Access101 } from '@react95/icons'

...
<Access101 />
...
```
is this not the most barebones usage? I have also updated my next.config.js as outlined in readme. 

```
/** @type {import('next').NextConfig} */

const withFonts = require('next-fonts');
const withTM = require('next-transpile-modules')([
  '@react95/core',
  '@react95/icons',
]);

const nextConfig = {
  reactStrictMode: true,
  swcMinify: true,
};

module.exports = withTM(withFonts(nextConfig));
```

**Comment by ggdaltoso** [comment, marked as answer]
Hi, @Nashtronaut and thanks for using @react95/core!

We have a [template for NextJS](https://github.com/React95/nextjs-template) that I've just updated to the lastest version!

Feel free to use!
